appveyor.yml

```diff
@@ -0,0 +1,15 @@
+---
+clone_depth: 10
+install:
+  - '"C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat" amd64'
+  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
+  - gem install rake-compiler
```

Comment:
I think we should extract to a rake task (I'm currently using a dirty workaround in ruby-openssl-docker: https://github.com/ruby/openssl/blob/f542fb60add7a7f9f57bd7097a5e61dc3e5e0a99/tool/ruby-openssl-docker/init.sh#L18)